src/database/migrations/20230615070344-add-password-and-iv.js

```diff
@@ -0,0 +1,58 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn(
+          "deployments", // name of Source model
+          "encrypted_password", // name of column we're adding
+          {
+            type: Sequelize.TEXT,
+            allowNull: true,
+            transaction: t,
+          }
+        ),
+        queryInterface.addColumn(
+          "deployments", // name of Source model
+          "encryption_iv", // name of column we're adding
+          {
+            type: Sequelize.TEXT,
+            allowNull: true,
+            transaction: t,
+          }
+        ),
+        queryInterface.addColumn(
+          "deployments", // name of Source model
+          "password_date", // name of column we're adding
+          {
+            type: Sequelize.DATE,
+            allowNull: true,
+            transaction: t,
+          }
```

Comment:
could i get more information/context on what `encryption_iv + password_date` implies? i was under the impression that we'd only need `encrypted_password`?

Reply:
`encryption_iv` is the initialisation vector used for encryption - this is needed so that similar patterns of text when encrypted don't give rise to similar ciphertexts! This information needs to be stored to retrieve the original unencrypted value

`password_date` is an additional field which will be helpful for a future feature - we eventually want to be able to tell when a repo password has not been changed for a certain amount of time, so this is useful for that, though it's currently unused